salalib/pointdata.cpp

```diff
@@ -1095,6 +1095,30 @@ void PointMap::outputConnectionsAsCSV(ostream& myout, std::string delim)
     }
 }
 
+void PointMap::outputLinksAsCSV(ostream& myout, std::string delim)
+{
+    myout << "RefFrom" << delim << "RefTo";
+    std::vector<PixelRef> seenPix;
+    for (int i = 0; i < m_cols; i++)
+    {
+        for (int j = 0; j < m_rows; j++)
+        {
+            if (m_points[i][j].filled() && m_points[i][j].m_node)
+            {
+                PixelRef mergePixelRef = m_points[i][j].getMergePixel();
+                if(mergePixelRef != NoPixel) {
+                    PixelRef pix(i,j);
+                    seenPix.push_back(pix);
+                    if(!(std::find(seenPix.begin(), seenPix.end(), mergePixelRef) != seenPix.end()))

```

Comment:
Couldn't we use something like a hashset for seenPix? This approach is very much in line with the flaky approach of converting PixRefs to ints to sort/compare them... Cf http://www.cplusplus.com/reference/unordered_set/unordered_set/